x/aol/client/cli/txRecord.go

```diff
@@ -2,94 +2,30 @@ package cli
 
 import (
 	"github.com/spf13/cobra"
-	"strconv"
 
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
 	"github.com/medibloc/panacea-core/x/aol/types"
 )
 
-func CmdCreateRecord() *cobra.Command {
+func CmdAddRecord() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-record [key] [value] [nanoTimestamp] [writerAddress]",
-		Short: "Creates a new record",
+		Use:   "add-record [ownerAddress] [topicName] [key] [value]",
+		Short: "Add a new record",
 		Args:  cobra.ExactArgs(4),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			argsKey := string(args[0])
-			argsValue := string(args[1])
-			argsNanoTimestamp, _ := strconv.ParseInt(args[2], 10, 64)
-			argsWriterAddress := string(args[3])
-
 			clientCtx, err := client.GetClientTxContext(cmd)
 			if err != nil {
 				return err
 			}
 
-			msg := types.NewMsgCreateRecord(clientCtx.GetFromAddress().String(), string(argsKey), string(argsValue), int32(argsNanoTimestamp), string(argsWriterAddress))
-			if err := msg.ValidateBasic(); err != nil {
-				return err
-			}
-			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
-		},
-	}
-
-	flags.AddTxFlagsToCmd(cmd)
-
-	return cmd
-}
-
-func CmdUpdateRecord() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "update-record [id] [key] [value] [nanoTimestamp] [writerAddress]",
-		Short: "Update a record",
-		Args:  cobra.ExactArgs(5),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			id, err := strconv.ParseUint(args[0], 10, 64)
+			feePayerAddress, err := cmd.Flags().GetString(flagFeePayer)
```

Comment:
Who is `feePayerAddress`?
It looks like not previouslycode

Reply:
For the `add-record` tx, we've adopted a 'fee payer' concept who pay fees on behalf of the 'writer':
https://github.com/medibloc/panacea-core/blob/985271f795c2ff0bf1f6b63152d97fabb41dbfa2/x/aol/client/cli/tx.go#L152

Reply:
I understand. Thanks